site/smart-accounts/accounts/light-account.md

```diff
@@ -12,41 +12,49 @@ head:
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
   - - meta
     - name: twitter:title
-      content: Light Account Smart Contract
+      content: Light Account
   - - meta
     - name: twitter:description
       content: Follow this guide to use Light Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337.
 ---
 
-# Light Account Smart Contract
+# Light Account
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
```

Comment:
i'd keep this tbh.

Reply:
it's not gone. down in ## Overview section, first sentence